cmd/create.go

```diff
@@ -189,30 +205,30 @@ var createCmd = &cobra.Command{
 			}
 		}()
 
+		// This calls the templates
 		err = project.CreateMainFile()
-
-		// once the create is done, stop the spinner
-		spinner.Quit()
-
-		// wait for the spinner to finish
-		wg.Wait()
 		if err != nil {
 			log.Printf("Problem creating files for project. %v", err)
 			cobra.CheckErr(textinput.CreateErrorInputModel(err).Err())
 		}
 
-		fmt.Println(endingMsgStyle.Render("\nNext steps cd into the newly created project with:"))
-		fmt.Println(endingMsgStyle.Render(fmt.Sprintf("• cd %s\n", project.ProjectName)))
+		fmt.Println(endingMsgStyle.Render("\nNext steps:"))
+		fmt.Println(endingMsgStyle.Render(fmt.Sprintf("• cd into the newly created project with: `cd %s`\n", project.ProjectName)))
+
+		if options.Advanced.Choices["AddHTMXTempl"] {
+			fmt.Println(endingMsgStyle.Render("• Install the templ cli if you haven't already by running `go install github.com/a-h/templ/cmd/templ@latest`\n"))
+			fmt.Println(endingMsgStyle.Render("• Generate templ function files by running `templ generate`\n"))
+		}
 
 		if isInteractive {
 			nonInteractiveCommand := utils.NonInteractiveCommand(cmd.Use, cmd.Flags())
 			fmt.Println(tipMsgStyle.Render("Tip: Repeat the equivalent Blueprint with the following non-interactive command:"))
 			fmt.Println(tipMsgStyle.Italic(false).Render(fmt.Sprintf("• %s\n", nonInteractiveCommand)))
-			err = tprogram.ReleaseTerminal()
-			if err != nil {
-				log.Printf("Could not release terminal: %v", err)
-				cobra.CheckErr(err)
-			}
+		}
+		err = spinner.ReleaseTerminal()
+		if err != nil {
+			log.Printf("Could not release terminal: %v", err)
```

Comment:
nit:
if we get here, cant we make sure to release the terminal for them? Maybe explicitly setting a kill for the terminal? Otherwise it will be hanging and the user needs to close the terminal.
Not blocking, just some food for thought

Reply:
What would that look like? Would we want to do an `os.Exit` or something like that?

Won't the `CheckErr` handle exiting with a non-zero exit code?

Reply:
if we pass a non-nill value to CheckErr it will run os.Exit(1)